internal/stanza/converter_test.go

```diff
@@ -208,15 +208,18 @@ func TestAllConvertedEntriesAreSentAndReceived(t *testing.T) {
 
 			converter := NewConverter(
 				WithWorkerCount(1),
-				WithMaxFlushCount(tc.maxFlushCount),
-				WithFlushInterval(10*time.Millisecond), // To minimize time spent in test
 			)
 			converter.Start()
 			defer converter.Stop()
 
 			go func() {
-				for _, ent := range complexEntries(tc.entries) {
-					assert.NoError(t, converter.Batch(ent))
+				entries := complexEntries(tc.entries)
+				for i := 0; i < tc.entries; i += int(tc.maxFlushCount) {
+					if i+int(tc.maxFlushCount) > tc.entries {
+						assert.NoError(t, converter.Batch(entries[i:tc.entries]))
+					} else {
+						assert.NoError(t, converter.Batch(entries[i:i+int(tc.maxFlushCount)]))
+					}
 				}
```

Comment:
Nit: This is a bit tricky to read. Perhaps a little clearer:
```suggestion
				for from := 0; from < tc.entries; from += int(tc.maxFlushCount) {
					to := from + int(tc.maxFlushCount)
					if to > tc.entries {
						to = tc.entries
					}
					assert.NoError(t, converter.Batch(entries[from:to]))
				}
```